zboxcore/sdk/sdk.go

```diff
@@ -882,16 +913,17 @@ func CreateAllocationWith(options CreateAllocationOptions) (
 		return CreateAllocationForOwner(options.Name, client.GetClientID(),
 			client.GetClientPublicKey(), options.DataShards, options.ParityShards,
 			options.Size, options.Expiry, options.ReadPrice, options.WritePrice, options.Lock,
-			options.BlobberIds)
+			options.BlobberIds, options.IsImmutable, options.ThirdPartyExtendable, options.FileOptionsParams)
```

Comment:
do we need immutable any more? we can use `0` flag for FileOptionsParams

Reply:
hmm. I agree with that. `IsImmutable` was there before I started working on NFT so honestly I didn't like to change it. I think this should also be mentioned in the documentation. Also `IsImmutable` is used in `0chain` and `blobber` as well so this will require some time to change. But to avoid the mess of having multiple related branches in different PRs, I'll remove it within the NFT PRs.

Reply:
Removed from all other PRs